src/main.c

```diff
@@ -38,9 +38,10 @@ void print_help(char *app_name) {
            "\t%s info\n"
            "\t%s version\n"
            "\t%s check <HOTP CODE>\n"
-           "\t%s regenerate <ADMIN PIN>\n"
+           "\t%s reset [ADMIN PIN]\n"
```

Comment:
You mean `[SECRET APP PIN]` ?

Reply:
> You mean `[SECRET APP PIN]` ?

I restate that UX with many Admin PIN if no User PIN counterpart is misleading.

Yes, Secret App PIN. At least for everything exposed to user and Heads as output.

Admin /User PIN makes sense when one have power over the other which is not the case here and users continuously lost with which PIN is asked here for security dongle, where Heads can only reuse PIN to simplify things.

Secret app pin has counter of 6 for that reason as opposed to user/Admin PIN of gpg if I understood correctly.

Do you understand what I'm talking about for sake of less support requests and UX consistency and less user confusion?

Yes, secret app PIN.

Reply:
> I repeat
> 
> You proposed :
> > See also issue #36 
> > Change this:
> > ```
> > 
> > HOTP code verification application, version 1.6
> > Connected device status:
> > 	Card serial: 0x7BE66C6C
> > 	Firmware: v4.13
> > 	Card counters: Admin 6, User 6
> > Operation success
> > ```
> > To
> > ```
> > HOTP code verification application, version 1.6
> > Connected device status:
> > 	Card serial: 0x7BE66C6C
> >         Firmware Nitrokey 3: v1.7.1
> > 	Firmware Secrets app: v4.13
> > 	Secret app pin counters : Admin 6, User 6
> > Operation success
> > ```
> 
> I proposed and restated:
> 
> **Even more sensical**: no secret app even named  anywhere because there is none on non nk3(regression), so no version of non existing secret app, no secret app pin, just real information :
> ```
> HOTP code verification application, version 1.7
> Connected device status:
> Card serial: 0x7BE66C6C
>          Firmware Nitrokey 2: v1.7.1
> OpenPGP smartcard PIN counters : Admin: 3, User: 3
> Operation success
> ```
> 
> For nk3:
> ```
> HOTP code verification application, version 1.7
> Connected device status:
> Card serial: 0x7BE66C6C
>          Firmware Nitrokey 3: v1.7.1
>          Firmware Secrets app: v4.13
> Secret app PIN counter : 6
> OpenPGP smartcard PIN counters : Admin: 3, User: 3
> Operation success
> ```
> 
> _Originally posted by @tlaurion in https://github.com/Nitrokey/nitrokey-hotp-verification/issues/38#issuecomment-2487450702_